packages/react-dom/src/events/__tests__/DOMModernPluginEventSystem-test.internal.js

```diff
@@ -1083,12 +1083,11 @@ describe('DOMModernPluginEventSystem', () => {
           }
         });
 
-        describe('ReactDOM.useEvent', () => {
+        describe.skip('ReactDOM.useEvent', () => {
```

Comment:
Intentionally skipping, as we'll bring these back with the new API and to avoid test churn on good tests I thought we'd leave them in for now.